packages/eslint-plugin/docs/rules/no-floating-promises.mdx

```diff
@@ -176,6 +176,44 @@ returnsSafePromise();
 </TabItem>
 </Tabs>
 
+### `allowForKnownSafePromiseReturns`
+
+This option allows marking specific functions as "safe" to be called to create floating Promises.
+For example, you may need to do this in the case of libraries whose APIs may be called without handling the resultant Promises.
+
+This option takes the same array format as [`allowForKnownSafePromises`](#allowForKnownSafePromises).
+
+Examples of code for this rule with:
+
+```json
+{
+  "allowForKnownSafePromiseReturns": [
+    { "from": "file", "name": "safe", "path": "input.ts" }
+  ]
+}
+```
+
+<Tabs>
+<TabItem value="❌ Incorrect">
+
+```ts option='{"allowForKnownSafePromiseReturns":[{"from":"file","name":"safe","path":"input.ts"}]}'
+declare function unsafe(...args: unknown[]): Promise<void>;
+
+unsafe('...', () => {});
+```
+
+</TabItem>
+<TabItem value="✅ Correct">
+
+```ts option='{"allowForKnownSafePromiseReturns":[{"from":"file","name":"safe","path":"input.ts"}]}' skipValidation
```

Comment:
> `skipValidation`

This was a fun one to discover:

* The playground uses `input.ts`
* `docs.test.ts` uses `file.ts`

I'm stepping out, but separate from this PR we should fix the two to be in sync.

Reply:
#9269 -> #9271